generate_nibetaseries_images.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
+
+set -e
+
+# Generate Dockerfile.
+generate_docker() {
+  docker run --rm kaczmarj/neurodocker:0.5.0 generate docker \
+    --base=neurodebian:stretch-non-free \
+    --install gcc g++ graphviz \
```

Comment:
added graphviz to build docs/to allow users to build the workflow graph (from nipype) if they so choose.

Reply:
Good point.